source/includes/_ios_targeted_sampling.md

```diff
@@ -0,0 +1,56 @@
+# Targeted Sampling
+
+```objective_c
+NSString *clientID = @"CLIENT_ID";
+NSString *accountToken = @"ACCOUNT_TOKEN";
+
+[Wootric configureWithClientID:clientID accountToken:accountToken];
+[Wootric setEndUserEmail:@"nps@example.com"];
+[Wootric setEndUserCreatedAt:@1234567890];
+[Wootric setEndUserProperties:@{ @"Example_string": @"pro_plan", @"Example_value": @12}];
+
+[Wootric showSurveyInViewController:self event:@"On Purchase Event"];
+```
+```swift
+let clientID = "CLIENT_ID"
+let accountToken = "ACCOUNT_TOKEN"
+
+Wootric.configure(withClientID: clientID, accountToken: accountToken)
+Wootric.endUserEmail = "nps@example.com"
+Wootric.endUserCreatedAt = NSNumber(value: 1234567890)
+Wootric.endUserProperties = [
+  "Example_string": "pro_plan",
+  "Example_value": NSNumber(value: 12)
+]
+
+Wootric.showSurvey(inViewController: self, event: "On Purchase Event")
+```
+```swift_three
+let clientID = "CLIENT_ID"
+let accountToken = "ACCOUNT_TOKEN"
+
+Wootric.configure(withClientID: clientID, accountToken: accountToken)
+Wootric.endUserEmail = "nps@example.com"
+Wootric.endUserCreatedAt = NSNumber(value: 1234567890)
+Wootric.endUserProperties = [
+  "Example_string": "pro_plan",
+  "Example_value": NSNumber(value: 12)
+]
+
+Wootric.showSurvey(inViewController: self, event: "On Purchase Event")
+```
+
+Targeted Sampling surveys can be triggered using a combination of
```

Comment:
What do you think about replacing this paragraph with this:

You can trigger surveys based on an end-user's property, an event that you might be tracking in your app or a combination of both. You must configure the rules that trigger these surveys inside Wootric Settings Panel.

Reply:
I like it 👍